README.md

```diff
@@ -5,7 +5,7 @@ This holds *all* of our [best practices](about.md).
 
 ## Git
 
-1. [Setup](git/setup.md)
+1. [Setup](git/00.setup.md)
```

Comment:
Hmm, yeah, that would be annoying, but I was thinking there's value in having an explicit ordering among the documents, so that things can build on other things (and assume a certain context, for brevity's sake) without people getting overly confused. Do you think the cost outweighs the value here?